ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiPackageManagementService.scala

```diff
@@ -138,6 +137,7 @@ private[apiserver] object ApiPackageManagementService {
       writeBackend: WritePackagesService,
       managementServiceTimeout: Duration,
       engine: Engine,
+      darReader: DarReader[Archive] = DarReader { case (_, x) => Try(Archive.parseFrom(x)) },
```

Comment:
Please extract this `darReader` out as a `lazy val DefaultDarReader` or something.